### DISCUSSION TITLE
Github-repos

### DISCUSSION THREAD

**Discussion by fodjouo** [learndevopsonline/learndevopsonline#1553] [Q&A]
He Raghu, 
   i created a Github-repos folder  just like you did. now i cant see when i try to have access to it when setting up intelliJ, what do i do. or should i ask what is the issue?

**Comment by prabhavith** [comment]
@fodjouo  After creating folder, right click and give **open git bash here**. And now access folder with intelliJ.

**Comment by fodjouo** [comment]
Thank you

**Comment by fodjouo** [comment]
i did all that and i still dont see the folder that i created in the in my
documents

On Tue, 31 Oct 2023 at 11:49, Prabha ***@***.***> wrote:

> @fodjouo <https://github.com/fodjouo> After creating folder, right click
> and give *open git bash here*. And now acess folder with intelliJ.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/learndevopsonline/learndevopsonline/discussions/1553#discussioncomment-7438153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYSEMXBNL7XXRQN4MZG4GB3YCE24HAVCNFSM6AAAAAA6X2S56KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMZYGE2TG>
> .
> You are receiving this because you were mentioned.Message ID:
> <learndevopsonline/learndevopsonline/repo-discussions/1553/comments/7438153
> @github.com>
>


-- 

*Sincerely, Jean-Pierre Fodjouo*
*(720) 252-0802*

**Reply by sreedharm07** [reply]
hi @fodjouo 
steps 
1 - create repo in git hub 
2 - copy repo code and go to our seperate devops folder where all repos were cloned. the git bash there
3 - git clone URL 
4 - Then in intillij open the folder where you cloned git, then you can see.

for you its not appearing in intillij may be because of you are clonning in one folder and in intillij you seeing in another folder.please check this

**Reply by fodjouo** [reply]
hello,
   i was able to get the process done but here is what i am having. i dont have the commit option, and as compared to the prof's screen, i dont have the 
Update project
commit
push the way he does

**Reply by sreedharm07** [reply]
It is may be because of you using different version raghu was using different version 
You can find them in another place for you please see here and there you can find

**Comment by fodjouo** [comment]
guys here is the new problem that i am encountering
[image: image.png]

On Tue, 31 Oct 2023 at 11:49, Prabha ***@***.***> wrote:

> @fodjouo <https://github.com/fodjouo> After creating folder, right click
> and give *open git bash here*. And now acess folder with intelliJ.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/learndevopsonline/learndevopsonline/discussions/1553#discussioncomment-7438153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYSEMXBNL7XXRQN4MZG4GB3YCE24HAVCNFSM6AAAAAA6X2S56KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMZYGE2TG>
> .
> You are receiving this because you were mentioned.Message ID:
> <learndevopsonline/learndevopsonline/repo-discussions/1553/comments/7438153
> @github.com>
>


-- 

*Sincerely, Jean-Pierre Fodjouo*
*(720) 252-0802*

**Reply by sreedharm07** [reply]
Hi @fodjouo this is the closed discussion
if you have a error or issue please rise new discussion.

**Comment by fodjouo** [comment]
why is is that i can't create a Workstation EC2 instance?


On Mon, 6 Nov 2023 at 17:16, Jean-Pierre Fodjouo ***@***.***> wrote:

> guys here is the new problem that i am encountering
> [image: image.png]
>
> On Tue, 31 Oct 2023 at 11:49, Prabha ***@***.***> wrote:
>
>> @fodjouo <https://github.com/fodjouo> After creating folder, right click
>> and give *open git bash here*. And now acess folder with intelliJ.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/learndevopsonline/learndevopsonline/discussions/1553#discussioncomment-7438153>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYSEMXBNL7XXRQN4MZG4GB3YCE24HAVCNFSM6AAAAAA6X2S56KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMZYGE2TG>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> <learndevopsonline/learndevopsonline/repo-discussions/1553/comments/7438153
>> @github.com>
>>
>
>
> --
>
> *Sincerely, Jean-Pierre Fodjouo*
> *(720) 252-0802*
>


-- 

*Sincerely, Jean-Pierre Fodjouo*
*(720) 252-0802*

**Reply by sreedharm07** [reply]
Hi @fodjouo 
you are creating multiple threads please use only one thread
